changelogs/fragments/snmp_server_fix.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - "`snmp_server` - fix host delete when authentication options are present."
```

This gets turned into rst which does strange things with single backticks. But we don't usually use them for the plugin anyway.
```suggestion
  - "snmp_server - fix host delete when authentication options are present."
```